marvel/main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"github.com/davecgh/go-spew/spew"
+	"github.com/joho/godotenv"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+)
+
+func main() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal("Error loading .env file")
+	}
+	log.Println(os.Getenv("MARVEL_PUBLIC_KEY"))
```

Comment:
Avoid printing out secrets to logs in your final code. It's find during dev locally, not okay in your final code as it presents a security risk.